modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -49,12 +56,26 @@ const applyCellData = (editor: Editor, cells: HTMLTableCellElement[], data: Cell
   const dom = editor.dom;
   const isSingleCell = cells.length === 1;
 
-  Arr.each(cells, (cell) => {
+  const table = TableLookup.table(SugarElement.fromDom(cells[0])).getOrNull();
+
+  const warehouse = Warehouse.Warehouse.fromTable(table);
```

Comment:
Ahh... I believe we may have exported `Warehouse` incorrectly, as this double `Warehouse.Warehouse` shouldn't be done.

Reply:
I figured as much. How do we correctly export it? I didn't see an immediate error.